server/lib/src/constants/uuids.rs

```diff
@@ -118,7 +118,7 @@ pub const _UUID_SCHEMA_ATTR_PRIMARY_CREDENTIAL: Uuid =
 pub const _UUID_SCHEMA_CLASS_PERSON: Uuid = uuid!("00000000-0000-0000-0000-ffff00000044");
 pub const _UUID_SCHEMA_CLASS_GROUP: Uuid = uuid!("00000000-0000-0000-0000-ffff00000045");
 pub const _UUID_SCHEMA_CLASS_ACCOUNT: Uuid = uuid!("00000000-0000-0000-0000-ffff00000046");
-// GAP - 47
+pub const UUID_SCHEMA_ATTR_NAMEHISTORY: Uuid = uuid!("00000000-0000-0000-0000-ffff00000133");
```

Comment:
Should be in numerical order in the file, so move this down to the bottom of the list IMO.

Reply:
Can you move this?